### PR TITLE
Feature/quickstart sample project

### PR DESCRIPTION
**Pull request by chunkhoang**

### CODE REVIEW

**Comment by charlychiu** [comment]
@chunkhoang This will also require adding pop-up ad sample code

File: README.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+```
+
+3. Sync your project.
+
+
+<a id="2-sdk-initialization"></a>
+### 2. SDK initialization
+
+<a id="a-set-bw-server"></a>
+#### a. Set the BW Server
```
Comment (charlychiu) [inline README.md]: We have already specified the BW server within the SDK, and we prefer that users do not make any changes to it

File: README.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
+```
+
+*   `ACCESS_COARSE_LOCATION` and `ACCESS_FINE_LOCATION` allow the device to send user location data for ad targeting, which can increase revenue by making impressions more valuable to buyers.
+*   `READ_PHONE_STATE` allows the Web View API to collect device data. 
+
+2. Configure your app by following the instructions for the [Google Mobile Ads SDK](https://developers.google.com/admob/android/quick-start#import_the_mobile_ads_sdk).
```
Comment (charlychiu) [inline README.md:139]: This seems to be specifically used only when installing via Maven distribution, correct?

Comment (chunkhoang) [reply README.md:139]: these permissions are needed when using WebViewAPI. They are required when installing via both methods (aar and Maven)

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+import com.bridgewell.bwmobile.ads.inapp.listener.BwsAdViewListener
+import com.bridgewell.bwmobile.ads.inapp.model.BwsAdView
+import com.bridgewell.quickstart.android.R
+import com.bridgewell.quickstart.android.activity.ui.adapter.TabbedViewAdapter
+import com.bridgewell.quickstart.android.data.AdType
+import com.bridgewell.quickstart.android.utils.showToast
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import koleton.api.hideSkeleton
+import koleton.api.loadSkeleton
```
Comment (charlychiu) [inline app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:21]: image

Is this being used?

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+        val selectedAdType = AdType.entries[selectedTabIndex]
+
+        for (i in 1..17) {
+            val imageViewId = resources.getIdentifier("imageView$i", "id", packageName)
+            val imageView = findViewById<ImageView>(imageViewId)
+            val skeletonColor = when (i) {
+                1, 2, 5, 6, 11, 10 -> R.color.skeleton_gray_1
+                else -> R.color.skeleton_gray_2
+            }
+            imageView.loadSkeleton {
```
Comment (charlychiu) [inline app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:86]: image
Does my IDE need any adjustments?

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+                    configID = CONFIG_ID_BANNER,
+                    refreshTimeSeconds = 0,
+                    listener = listener
+                )
+            }
+            AdType.STICKY_BOTTOM_RIGHT -> {
+                inAppApi.createBwsRightSideStickyAd(
+                    this,
+                    configID = CONFIG_ID_BANNER,
+                    bottomMargin = 77,
```
Comment (charlychiu) [inline app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt]: Cannot find a parameter with this name: bottomMargin

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+                    }
+                }
+                val adWrapperView = findViewById<FrameLayout>(R.id.frameAdWrapper)
+                inAppApi.createBwsBannerAd(
+                    this,
+                    configID = CONFIG_ID_BANNER,
+                    width = adWrapperView.width,
+                    height = adWrapperView.height,
+                    viewContainer = adWrapperView,
+                    listener = listener
```
Comment (charlychiu) [inline app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:180]: image

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt (hunk 3 above)
Comment (chunkhoang) [reply app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:21]: Yes it's being used by the loading animation of the skeleton

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt (hunk 4 above)
Comment (chunkhoang) [reply app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:86]: Yes, please sycn the project. Gradle files has changed

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt (hunk 6 above)
Comment (chunkhoang) [reply app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt:180]: This issue will be resolved after the project is synced

File: app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt (hunk 5 above)
Comment (chunkhoang) [reply app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt]: in the new Figma design, the bottom margin changed. Add an optional parameter to the BWSDK to be able to adjust the bottom margin of bottom-right sticky banner. Please check my new PR

**Pull request merged**